crates/cheatcodes/src/evm.rs

```diff
@@ -76,6 +81,72 @@ pub struct DealRecord {
     pub new_balance: U256,
 }
 
+/// Storage slot diff info.
+#[derive(Serialize, Default)]
+#[serde(rename_all = "camelCase")]
+struct SlotStateDiff {
+    /// Initial storage value.
+    previous_value: B256,
+    /// Current storage value.
+    new_value: B256,
+}
+
+/// Balance diff info.
+#[derive(Serialize, Default)]
+#[serde(rename_all = "camelCase")]
+struct BalanceDiff {
+    /// Initial storage value.
+    previous_value: U256,
+    /// Current storage value.
+    new_value: U256,
+}
+
+/// Account state diff info.
+#[derive(Serialize, Default)]
+#[serde(rename_all = "camelCase")]
+struct AccountStateDiffs {
+    /// Address label, if any set.
+    label: Option<String>,
+    /// Account balance changes.
+    balance_diff: Option<BalanceDiff>,
+    /// Placeholder for nonce diff, to be implemented.
+    nonce_diff: Option<(U256, U256)>,
```

Comment:
I think that until it is implemented we should not include the `nonce_diff` as it could lead to incorrect conclusions (assuming that the nonce did not change (`null`) when it in fact did)

Reply:
:+1: backed off with https://github.com/foundry-rs/foundry/pull/9435/commits/b2ae150e4d503f0a843fa195eca815c249e26438 agree, don't think will be breaking change if we introduce a new field in json output